app/src/main/java/org/oppia/app/audioplayer/AudioPlayerFragmentController.kt

```diff
@@ -0,0 +1,81 @@
+package org.oppia.app.audioplayer
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import android.widget.PopupMenu
+import android.widget.SeekBar
+import androidx.fragment.app.Fragment
+import javax.inject.Inject
+import kotlinx.android.synthetic.main.audioplayer_fragment.view.*
+import org.oppia.app.fragment.FragmentScope
+import org.oppia.app.R
+import org.oppia.domain.AudioPlayerController
+
+@FragmentScope
+class AudioPlayerFragmentController @Inject constructor(
```

Comment:
Nit: this should be a 'Presenter' not a 'Controller'.

Also, how should this interop with the one Rajat is introducing in #105? Is it still needed with that PR?

Reply:
I think this will replace the one in #105, the only thing I would need to add is opening the LanguageDialog